database.js

```diff
@@ -20,7 +20,7 @@ async function addUser(Email, username, password){
     let collection = db.collection("web-app");
     let response = await collection.find({email:Email}).toArray();
     if (response.length > 0) return "USER ALREADY EXISTS!";
-    else await collection.insertOne({ email: Email, user: username, pass: password  });
+    else await collection.insertOne({ email: Email, user: username, pass: password  }); //maybe use bcrypt here like pass: hashedpassword
```

Comment:
I already hashed it (this is hashed password)
![image](https://github.com/anuragdaksh7/web-based-expense-tracker/assets/84393491/daa86d87-db67-4c18-872c-ad9dc5241095)
- ss from the DB

Reply:
wow i didnt know that it could be hashed fram a db

Reply:
```javascript

        const { createHash } = require('crypto');

        function hash(string) {
             return createHash('sha256').update(string).digest('hex');
        }
        console.log(hash(password))

``` 

I used this to hash in app.js